### ISSUE TITLE
checkpoint and learning rate/dataset

### ISSUE THREAD

**Issue by fjt0324** [ShiWuxuan/SUFT#1]
Hi, thanks for your excellent job. Several problems to bother you:
1. Are the default parameters in the code the ones that produce the result in paper? For example, in the paper,"The initial learning rate is 1𝑒−4 and reduced by 0.1 for every 100 epochs.", but the lr is decayed in epoch 50 and epoch 100 as in the code
2. Is trained checkpoint available?
